tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
+envlist = flake8, coverage, docs, py36
```

Comment:
Do we want to support other 3.x versions?

Reply:
Yes, but I want to do that once we have CI working because I only have 3.6 on my computer right now.